examples/23-kubeflow-spot-instance.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   # Name of your cluster, change to whatever you find fit.
   # If changed, make sure to change all nodegroup tags from 
   # 'k8s.io/cluster-autoscaler/kubeflow-us-east-1: "owned"' --> 'k8s.io/cluster-autoscaler/your-new-name: "owned"'
-  name: kubeflow-us-east-1
+  name: cluster-23
```

Comment:
https://github.com/weaveworks/eksctl/blob/15864fce2df02e9b8215af80a0b9a778d960da27/examples/23-kubeflow-spot-instance.yaml#L20

The following tags (there are a few)
https://github.com/weaveworks/eksctl/blob/15864fce2df02e9b8215af80a0b9a778d960da27/examples/23-kubeflow-spot-instance.yaml#L54

Reply:
bleh